src/main/java/com/bullhornsdk/data/model/entity/core/standard/JobSubmission.java

```diff
@@ -27,7 +27,7 @@
     "customText21", "customText22", "customText23", "customText24", "customText25",
     "customTextBlock1", "customTextBlock2", "customTextBlock3", "customTextBlock4", "customTextBlock5",
     "dateAdded", "dateLastModified", "dateWebResponse", "endDate", "isDeleted", "isHidden", "jobOrder", "jobSubmissionCertificationRequirements",
-    "migrateGUID", "owners", "payRate", "salary", "sendingUser", "source", "startDate", "status", "tasks"})
+    "migrateGUID", "owners", "payRate", "salary", "sendingUser", "source", "startDate", "status", "tasks", "latestAppointment"})
```

Comment:
@scriptom latestAppointment needs to be an actual field on the POJO, no? This would just be adding it to the propertyOrder